ops/demo-deploy.tmpl.yaml

```diff
@@ -74,6 +74,8 @@ extraEnvVars: &envVars
     value: $DB_PASSWORD
   - name: DB_USER
     value: main
+  - name: EXTERNAL_IIIF_URL
```

Comment:
Do we need to include these values in staging-deploy.tmpl.yaml too @orangewolf ?